src/event.lib.php

```diff
@@ -722,4 +722,54 @@ public static function setNewConfigKeys()
 
         return $resp;
     }
+
+    /**
+     * Clear last logins or failed login logs.
+     * 
+     * This can also be done via Sucuri Security -> Settings -> Data Storage,
+     * however to improve the user experience, a button on Last Logins  and on
+     * Failed logins sections was added and it triggers the removal of
+     * sucuri/sucuri-lastlogins.php and sucuri/sucuri-failedlogins.php.
+     * 
+     * @param string $filename Name of the file to be deleted.
+     *
+     * @return HTML Message with the delete action outcome.
+     */
+    public static function clearLastLogs($filename)
+    {
+        // Get the complete path of the file.
+        $filepath = SucuriScan::dataStorePath($filename);
+
+        // Do not proceed if not possible.
+        if (
+            !is_writable(dirname($filepath)) ||
+            !file_exists($filepath) ||
+            is_dir($filepath) ||
+            empty($filepath)
+        ) {
+            return SucuriScanInterface::error(
+                sprintf(
+                    __('%s is empty or can not be deleted.', 'sucuri-scanner'),
```

Comment:
'can not' --> 'cannot'.

If the file is empty or doesn't exist, there's no need to show an error message. Or, if you insist on showing an error message if the file is empty, then I suggest to separate it from the case where the file cannot be deleted (2 separate messages).